io.catenax.part_type_information/1.0.0/PartTypeInformation.ttl

```diff
@@ -0,0 +1,148 @@
+   samm:characteristic :partSitesInformationAsPlannedCharacteristic.
+
+:PartTypeInformationCharacteristic a samm:Characteristic;
+   samm:preferredName "Part Type Information Characteristic"@en;
+   samm:description "The characteristics of the part type"@en;
+   samm:dataType :PartTypeInformationEntity.
+
+:partSitesInformationAsPlannedCharacteristic a samm-c:Set;
+   samm:preferredName "Part Sites Information as Planned Characteristic"@en;
+   samm:description "The characteristic of the part site information that represents a set of possible site to a respective part. "@en;
+   samm:dataType :partSitesInformationAsPlannedEntity.
+
+:PartTypeInformationEntity a samm:Entity;
+   samm:preferredName "Part Type Information Entity"@en;
+   samm:description "Encapsulation for data related to the part type"@en;
+   samm:properties(:manufacturerPartId :nameAtManufacturer [ samm:property ext-classification:partClassification; samm:optional true ]).
+
+:partSitesInformationAsPlannedEntity a samm:Entity;
+   samm:preferredName "Part Sites Information as Planned Entity"@en;
+   samm:description "Describes the ID, function and validity date of a site for the associated part in the AsPlanned context."@en;
+   samm:properties(:catenaXsiteId :function [ samm:property :functionValidFrom; samm:optional true ] [ samm:property :functionValidUntil; samm:optional true ]).
+
+:manufacturerPartId a samm:Property;
+   samm:preferredName "Manufacturer Part ID"@en;
+   samm:description "Part ID as assigned by the manufacturer of the part. The Part ID identifies the part in the manufacturer`s dataspace. The Part ID references a specific version of a part. The version number must be included in the Part ID if it is available.\n\nThe Part ID does not reference a specific instance of a part and must not be confused with the serial number."@en;
+   samm:characteristic :PartIdCharacteristic;
+   samm:exampleValue "123-0.740-3434-A".
+
+:nameAtManufacturer a samm:Property;
+   samm:preferredName "Name at manufacturer"@en;
+   samm:description "Name of the part as assigned by the manufacturer"@en;
+   samm:characteristic :PartNameCharacteristic;
+   samm:exampleValue "Mirror left".
+
+:catenaXsiteId a samm:Property;
+   samm:preferredName "Catena-X site identifier"@en;
+   samm:description "The identifier of the site according to Catena-X BPDM. The catenaXsiteId must be a valid Catena-X BPN. The BPN is a unique, unchangeable identifier for Business Partners / company locations from foundation to closure, regardless of the different business relationships / structures between or within the Business Partners or company locations."@en;
+   samm:characteristic ext-number:BpnsTrait;
+   samm:exampleValue "BPNS1234567890ZZ".
+
+:function a samm:Property;
+   samm:preferredName "Function"@en;
+   samm:description "The function of the site in relation to the part(i.e. the activity within the value chain of the part that is performed at the site)"@en;
```

Comment:
missing space after part (relation to the **part(i.e.** the activity)
dot at the end of the sentence